### PR TITLE
Feature/add craft form

### PR DESCRIPTION
**Pull request by Wperricone**
## What was changed: 
- The form to add a new item to Craft Circle was given functionality and style. 

## Explain the reason for changes:
- The form is there so any user can add a new craft item to their user profile. 
- Used dev tools to make sure state is updated as a user types in the input
- Future update: link the submit button to the data

## Link to issue:
Updates # 11 and # 19

### CODE REVIEW

File: src/Components/Form.js [hunk 1]
```diff
@@ -1,18 +1,99 @@
...
+import React, { useState } from "react";
 import styled from 'styled-components';
+import colors from '../constants/colors';
+import Button from './Button';
 
 const Form = () => {
+  const [itemName, setItemName] = useState("")
+	const [amount, setItemAmount] = useState(0)
+	const [description, setItemDescription] = useState("")
+	const [category, setItemCategory] = useState("")
```
Comment (stephanie-roe) [inline src/Components/Form.js:10]: Great implementation of hooks!!!

File: src/Components/Form.js [hunk 2]
```diff
@@ -1,18 +1,99 @@
...
+`
+
+const CategoryAmountContainer = styled.div`
+display: flex;
+justify-content: space-between;
+width: 70%;
+`
+
+
+// potentially add a type to the form? would be useful if we choose to filter
```
Comment (stephanie-roe) [inline src/Components/Form.js:98]: great idea- we should take a look at implementing a button group to add this type.

**Review by stephanie-roe — APPROVED** [review]
looks good! now time to get that post up and running

**Comment by JoeRupp** [comment]
Nice work! Up next for this page is connecting the submit button to actually make a craft item.

**Pull request merged**